Source/WebCore/platform/graphics/skia/GraphicsContextSkia.cpp

```diff
@@ -534,9 +539,12 @@ void GraphicsContextSkia::setLineCap(LineCap lineCap)
     m_skiaState.m_stroke.cap = toSkiaCap(lineCap);
 }
 
-void GraphicsContextSkia::setLineDash(const DashArray&, float /*dashOffset*/)
+void GraphicsContextSkia::setLineDash(const DashArray& dashArray, float dashOffset)
```

Comment:
Could we do move semantics here?

```suggestion
void GraphicsContextSkia::setLineDash(DashArray&& dashArray, float dashOffset)
```